packages/openapi-v3/src/controller-spec.ts

```diff
@@ -270,12 +274,6 @@ export function jsonToSchemaObject(jsonDef: JsonDefinition): SchemaObject {
         result.allOf = _.map(json.allOf, item => jsonToSchemaObject(item));
         break;
       }
-      case 'definitions': {
```

Comment:
Please preserve the conversion here as we've discussed

Reply:
Talked with @shimks , the broken scenario will be:

```js
@model
class Foo {
   @property
   bar: Bar;
   @property
   baz: Baz;
};

@model
class Bar {
  @property
   name: string
};

@model
class Baz {
  @property
   name: string
};

class FooController {
  async function create(@param(<a_param_spec>) foo: Foo): void { }
}
```

Reply:
Addressed it in my PR.